asrtoolkit/data_handlers/stm.py

```diff
@@ -14,6 +14,11 @@
 from asrtoolkit.data_structures.segment import segment
 
 
+def footer():
+    " Returns footer with trailing line break "
```

Comment:
Usually, docstrings use 3 `"`.

```suggestion
    """ Returns footer with trailing line break """
```